frontend/installation-ui/web/micro-ui-internals/packages/modules/pm/src/pages/employee/CreateFieldPlan.js

```
@@ -910,36 +933,80 @@ const CreateFieldPlan = () => {
+  const handleBackNavigation = () => {
     switch (currentKey) {
       case 1:
-        setShowBackAlert(false);
-        window.history.back();
+        const savedFieldPlanDetails = {
+          districts: persistedFormData?.fieldPlanDetails?.districts,
+          blocks: persistedFormData?.fieldPlanDetails?.blocks,
+          fieldPlanDuration: persistedFormData?.fieldPlanDetails?.fieldPlanDuration,
+          healthFacilitiesCount: persistedFormData?.fieldPlanDetails?.healthFacilitiesCount,
+          activities: persistedFormData?.fieldPlanDetails?.activities,
+        };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)